<launch>

  <!--- Teleop Joystick -->
  <node pkg="pr2lite_moveit_config" type="pr2lite_teleop.py" name="pr2lite_teleop"/>
  <node pkg="joy" type="joy_node" name="joystick"/>
<!--
  <node pkg="pr2lite_moveit_config" type="festival_launcher.py" name="festival_launcher"/>
-->


</launch>
